### PR TITLE
Update Startup Career Fair info

### PR DESCRIPTION
**Pull request by NWChen**
Change date to 10/27 and stats for last year's SCF.

### CODE REVIEW

**Comment by RaymondXu** [comment]
fyi secrets file still visible

File: @ [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Folders
+/app/static/css/gen
+/app/static/img/uploaded/
+/lib
+/local
+/log/*
+/bin
+/include
```
Comment (alanhdu) [inline @]: What is this file?

**Comment by alanhdu** [comment]
Feel free to merge after fixing the conflict.

**Comment by NWChen** [comment]
Resolved merge conflicts with [@f769e1f27c20ff3c23df43b1d1a09c7f418e09a8](https://github.com/adicu/adi-website/commit/f769e1f27c20ff3c23df43b1d1a09c7f418e09a8). @melsawyer or @RaymondXu I don't have write access - could you review and/or merge in?

**Pull request merged**